docs/usage.rst

```diff
@@ -3,9 +3,18 @@ Usage
 =====
 
 Particles are a key component when dealing with decays.
-Refer to the [particle package](https://github.com/scikit-hep/particle) for how to deal with particles and PDG identification codes.
+Refer to the `particle package`_ for how to deal with particles and PDG identification codes.
```

Comment:
We decided to name the package `Particle` - your preference ;-). Otherwise all good.

Reply:
```suggestion
Refer to the `Particle package`_ for how to deal with particles and PDG identification codes.
```